addons/point_of_sale/static/tests/tours/OrderManagementScreen.tour.js

```diff
@@ -108,7 +108,14 @@ odoo.define('point_of_sale.tour.OrderManagementScreen', function (require) {
     OrderManagementScreen.check.orderDetailsHas({
         lines: [{ product: 'Whiteboard Pen', quantity: '5' }],
         total: '30',
+        customerNote: 'Customer Note',
```

Comment:
here you name it customerNote, but signature of function is 'note'

Reply:
Indeed, I just learned that recently